extensions/llama/python/llama.py

```diff
@@ -95,25 +105,43 @@ async def _run_inference_helper(self, model_input, options) -> List[Output]:
             except TypeError:
                 texts = [response["choices"][0]["text"]]
 
+            output_data_value : str = ''
+            last_response = texts[-1]
+            if isinstance(last_response, str):
+                output_data_value = last_response
+            else:
+                raise ValueError(f"Output {last_response} needs to be of type 'str' but is of type: {type(last_response)}")
```

Comment:
Same here. Is this all needed?

Reply:
Same comments as above, I can remove if we want